test/unit/specs/components/staking/PageValidator.spec.js

```diff
@@ -101,11 +107,11 @@ describe(`PageValidator`, () => {
       selfBond: 10
     }
     wrapper.setData({ validator })
-    expect(wrapper.vm.selfBond).toBe(`1000.00`)
+    expect(wrapper.vm.selfBond).toBe(`1000.00%`)
```

Comment:
Sounds strange 1000 %, is that expected?

Reply:
good call! will fix.